src/pages/signIn/SignIn.tsx

```diff
@@ -10,10 +10,9 @@ import { config } from "./config";
 
 import { Fields, ValidationFields, ConfigItem } from "./types";
```

Comment:
I think better create file ```intex.ts``` in folder ```types``` and export there all you types, and after that you can import your type like that ```import * as types from "./types";```